incentivizing_rewarding_participants.adoc

```diff
@@ -1,87 +1,129 @@
+
+Note that this reward model involves incentives with far-reaching and potentially irreversible consequences, as rescinding rights and privileges that you've granted to contributors can be difficult.
 
 === Sponsorships
 
-{Sponsored travel and/or attendance at community and industry events.}
+Sponsoring contributors' travel to (even accommodations at) industry events or professional conferences is another way to reward participation in a project.
+As project members deepen their investments in a project and community, they will frequently seek opportunities to speak about the project in these contexts—and to meet, share with, and learn from contributors from other projects.
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-Suspendisse viverra id leo et lacinia.
-Maecenas convallis lacinia diam, et lobortis quam tincidunt hendrerit.
-Mauris eget nulla at dolor ullamcorper lacinia.
+This reward model is therefore especially appropriate when you know that a contributor is motivated by a desire to expand a professional network.
+It's a way to invest in contributors' personal and professional development, recognize that their participation in the project is part of their overall plan for self-improvement, and invite them to assume a more public-facing role assisting project growth.
 
-=== Community designations
-
-{Access to new community roles, rights, and access privileges.}
-
-Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-Aliquam tempus ornare metus nec accumsan.
-Nulla et dui arcu.
-Proin hendrerit nibh velit, quis aliquet est interdum sit amet.
-Pellentesque vitae turpis tempor ex pulvinar sodales ut vitae arcu.
+Of all the reward models we've surveyed so far, this one likely requires the most resources.
+Simply put: it can be monetarily expensive.
+It's also a labor-intensive reward model, as it often requires complicated logistical work (purchasing, scheduling, coordinating, etc.) on behalf of project maintaners or other leaders.
 
 == Conclusion
 
-{This section reviews primary themes and concludes the chapter.}
-
-Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-Suspendisse viverra id leo et lacinia.
-Maecenas convallis lacinia diam, et lobortis quam tincidunt hendrerit.
-Mauris eget nulla at dolor ullamcorper lacinia.
+This chapter has explored the importance of community reward models, strategies for acknowledging and honoring various contributions to an open source project.
+Community managers and architects construct compelling community experiences when they help communities implement reward models that resonate with contributors' motivations for participating in a project.
+This chapter also described multiple reward models and discussed benefits of drawbacks of each.
+The more reward models a community can feature, the greater its chances of channeling enthusiastic participation from contributors with various backgrounds, motives, and talents.
```

Comment:
I don't completely disagree, but I would like to see more support for this statement.

Reply:
No sweat. I will augment this sentence to emphasize the connection between a diversity of reward models, participant motivations, and a multiplicity of incentives.